web/utils.py

```
@@ -287,7 +287,9 @@ def get_leaderboard(current_user=None, period=None, limit=10):
     else:  # Global leaderboard
         # Get global leaderboard
         leaderboard_entries = (
-            Points.objects.filter(user__profile__is_teacher=False, user__profile__is_profile_public=True)
+            Points.objects.filter(
+                user__profile__is_teacher=False, user__profile__is_profile_public=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)